hydrus/tests/conftest.py

```diff
@@ -46,9 +46,13 @@ def gen_dummy_object(class_title, doc):
     for class_path in doc.parsed_classes:
         if class_title == doc.parsed_classes[class_path]["class"].title:
             for prop in doc.parsed_classes[class_path]["class"].supportedProperty:
-                if isinstance(prop.prop, HydraLink) or prop.write is False:
+                if prop.write is False:
                     continue
-                if "vocab:" in prop.prop:
+                if isinstance(prop.prop, HydraLink):
```

Comment:
Again treat hydra:link as just links, not as resources.

Reply:
So I should treat `hydra:link` as just a normal column in our new schema right? Because as you had said that the burder for `hydra:link` falls on the client then